src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -4966,7 +5594,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         llvm::Value *idx = tmp;
         this->visit_expr_wrapper(x.m_arg, true);
         llvm::Value *str = tmp;
-        tmp = lfortran_str_item(str, idx);
+        tmp = lfortran_str_copy(str, idx, idx);
```

Comment:
to be reverted (https://github.com/lcompilers/lpython/commit/2f45421c06c2bee6a9918ae71ca0b88530ebd6fd).